### PR TITLE
add spec-url for `MerchantValidationEvent`

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

the spec is at https://w3c.github.io/merchant-validation/, which is deprecated but not remove from standard

### CODE REVIEW

**Review by teoli2003 — APPROVED** [review]
👍

**Review by teoli2003 — CHANGES_REQUESTED** [review]
Prior to land this we have to understand why Merchant-validation is not listed in w3c/browser-specs

**Comment by teoli2003** [comment]
I guess we should set the _deprecated_ flag to true and the _standard_ to false. What do you think @queengooborg ?

**Comment by skyclouds2001** [comment]
image

I check again, I guess this suggest it is intended to suugest that this is not a part of standard anymore

**Comment by skyclouds2001** [comment]
see https://github.com/w3c/browser-specs/issues/1162#issuecomment-1869205656 and https://github.com/w3c/browser-specs/pull/1163#issue-2055966690, it seems that this is deprecated, but, still a part of standard

**Comment by tidoust** [comment]
> but, still a part of standard

Not sure what you would call "a part of standard" and what that status may entail for BCD, but a Note is purely informative and not what W3C would call a standard. The Note merely exists to document how the feature was envisioned before it got removed from the actual standard.

**Comment by teoli2003** [comment]
Thanks @tidoust.

I think we shouldn't merge this, but let's have another opinion before. cc/ @Elchi3 @queengooborg

**Comment by queengooborg** [comment]
Yeah, if you look at the CI test, you'll notice that the spec URL is unknown.  Since we defer definition of what specs are standards track to `browser-specs`, and this spec URL isn't present in `browser-specs`, we shouldn't add them in or call them standard.

**Pull request closed without merging**
